### ISSUE TITLE
no-make installer does not install shFlags/src/shflags 

### ISSUE THREAD

**Issue by rdm** [nvie/gitflow#190]
windows 7, cygwin, after running current version of https://github.com/nvie/gitflow/raw/develop/contrib/gitflow-installer.sh

$ git flow init
C:/cygwin/usr/local/bin/gitflow-shFlags: line 1: shFlags/src/shflags: No such file or directory
C:\cygwin\usr\local\bin\git-flow: line 80: FLAGS: command not found
fatal: 'flow' appears to be a git command, but we were not
able to execute it. Maybe git-flow is broken?

$ cat /usr/local/bin/gitflow-shFlags
shFlags/src/shflags

$ ls $HOME/gitflow/shFlags

$ 

In other words, not only did the installer lose track of where gitflow was cloned to, but it will depend on ../src/shflags which does not exist in this context.

**Comment by jyr** [comment]
I have the same bug.

brew install git-flow
==> Cloning https://github.com/nvie/gitflow.git
Updating /Users/jyr/Library/Caches/Homebrew/git-flow--git
==> Checking out tag 0.4.1
Submodule 'shFlags' () registered for path 'shFlags'
==> make prefix=/usr/local/Cellar/git-flow/0.4.1 install
==> Cloning https://github.com/bobthecow/git-flow-completion.git
Updating /Users/jyr/Library/Caches/Homebrew/git-flow-completion--git
==> Checking out tag 0.4.1.0
/usr/local/Cellar/git-flow/0.4.1: 13 files, 132K, built in 7 seconds

$ git flow
fatal: 'flow' appears to be a git command, but we were not
able to execute it. Maybe git-flow is broken?

**Comment by MichaelPereira** [comment]
I have the same issue on windows 7 with cygwin

**Comment by faern** [comment]
I had the same problem. I had my friend email his copy to me and it works. Strange solution but I could not find the correct content somewhere else. I host the file here if anyone is interested:
<Sorry, hosting removed>

**Comment by ilcavero** [comment]
Thanks faern, I had the same issue windows7/cygwin and I was able to fix it by replacing my gitflow-shFlags with your hosted file

**Comment by rhbecker** [comment]
I encountered the same issue (Win7+cygwin). I basically used the same solution as suggested above, except I used what I found in nvie's submodule, here on github: https://github.com/nvie/shFlags/blob/master/src/shflags

**Comment by codedcontainer** [comment]
@ilcavero Your link is not longer working.

**Comment by tiagopotencia** [comment]
Hello! I had a problem like this! I've solved changing line 83 of git-flow file:

`#use the shFlags project to parse the command line arguments	. "$GITFLOW_DIR/gitflow-shFlags"`

to the value of `gitflow-shFlags`

`# use the shFlags project to parse the command line arguments
	. "$GITFLOW_DIR/shFlags/src/shflags"`

**Comment by nvie** [comment]
Closing all issues as I'm archiving this repo. git-flow lives on at [git-flow-next](https://github.com/gittower/git-flow-next) if you're still using it. Thanks for reporting!
